src/content/docs/extensions/vector.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: "Vector search"
+---
+
+The `VECTOR` extension provides an on-disk [HNSW](https://en.wikipedia.org/wiki/Hierarchical_navigable_small_world)-based vector index for accelerating similarity search over float array columns in tables.
+
+:::caution[Experimental Feature]
```

Comment:
Can we replace the term "experimental" with a more gentle term? Saying its experimental makes it seem it's not to be trusted. I propose that we don't make this a callout box to call attention to this.

We can simply mention this as a regular sentence and the bullets after it. See my suggestion below.